index.js

```diff
@@ -387,7 +387,7 @@ function compare(buffer1, start1, buffer2, start2) {
       buffer1.readInt32LE(start1 + len1) - buffer2.readDoubleLE(start2 + len2)
     )
 
-  if (type1 === DOUBLE && type1 === INT)
+  if (type1 === DOUBLE && type2 === INT)
```

Comment:
This also seemed like a mistake. Read carefully just to be sure.

Reply:
Yep, the return line below it makes it clear that this looks good